js/jquery.plugincreator.es6

```diff
@@ -0,0 +1,290 @@
+/*!
+ * jQuery Plugin Creator 0.3.0
+ * https://github.com/OOPMan/jquery-plugin-creator
+ *
+ * Copyright 2014 Adam Jorgensen
+ * Released under the MIT license.
+ * http://github.com/OOPMan/jquery-plugin-creator/LICENSE
+ *
+ */
+/**
+ * TODO: Add checks to prevent over-writing plugins
+ * TODO: Add ability to not specify plugin name and have a randomly generated name assigned
+ */
+"use strict";
+
+import esprima from "esprima";
+
+var noOp = jQuery.noop,
```

Comment:
@fluffels Rather use `let` here as all of the items in this `var` list are intended to be constants...